sqlalchemy_history/builder.py

```diff
@@ -174,7 +174,10 @@ def enable_active_history(self, version_classes):
         """
         for cls in version_classes:
             for prop in sa.inspect(cls).iterate_properties:
-                getattr(cls, prop.key).impl.active_history = True
+                if hasattr(cls, prop.key):
+                    # Mapper Arg `_sa_polymorphic_on` appear as columnproperty but is not attr of cls
+                    # Giving AttrError
+                    getattr(cls, prop.key).impl.active_history = True
```

Comment:
We should check as to why active_history = True is not required for implicit properties like policymorphic_on